### ISSUE TITLE
Use a different API, maybe?

### ISSUE THREAD

**Issue by emanuelfeld** [evonfriedland/geocoder#4]
I somewhat sloppily switched over to a Google basemap (commenting out the Leaflet bits) to address Google API ToS issues. I'd rather just dump the Google API altogether, but haven't found any that:
1. Don't require a key
2. Are adept at address parsing
3. Are free and open

OSM Nominatim largely fails the second (or, perhaps, is just missing a number of addresses). Pelias might be an option soon, since they're working on the second issue.

**Comment by blueogive** [comment]
I'm not acquainted with OSM Nominatim or Pelias, but you might 
consider the [Here API](https://developer.here.com/rest-apis/documentation/geocoder). The ToS are more generous than Google's, and I've had good luck using it with DC addresses. The API I'd **like** to be able to recommend is that of the [Data Science Toolkit](http://www.datasciencetoolkit.org/developerdocs#googlestylegeocoder), which is patterned after Google's API but without the ToS issues. Unfortunately, in my experience, it tends to return coordinates for non-matching addresses with no indication it is doing so.

HTH

**Comment by emanuelfeld** [comment]
@blueogive unfortunately, I don't believe Here is free ([plans](https://developer.here.com/get-started#/10134037)) and it requires a key for the application. Google's fine for now, but figured I'd flag this issue in case something better comes along.

**Comment by blueogive** [comment]
You're correct. Apologies. Only a couple months ago there was a free tier w/ 10K transactions/day and no key.

**Comment by MiniCodeMonkey** [comment]
Feel free to take a look at http://geocod.io too - we support batch geocoding.
